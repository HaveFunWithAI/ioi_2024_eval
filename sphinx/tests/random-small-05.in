49 450
24 20 0 0 15 32 39 34 32 14 41 37 38 27 3 48 7 6 35 18 23 33 14 14 9 32 20 30 10 32 24 44 11 47 36 4 23 33 23 34 38 39 4 37 3 6 40 7 20
2 29
2 23
2 48
44 48
34 44
32 34
3 48
15 29
27 32
3 19
32 37
27 36
21 29
9 32
41 48
0 36
2 39
16 23
12 37
6 15
23 30
14 39
11 44
35 36
7 19
3 13
22 29
21 26
19 31
11 18
20 44
4 44
1 13
32 33
33 45
15 46
41 47
34 40
17 29
15 24
12 28
39 43
28 42
5 43
10 45
24 38
25 27
8 17
16 42
4 11
21 38
10 30
1 37
10 29
19 45
5 34
1 44
28 43
19 23
4 46
23 41
1 35
16 28
11 16
22 26
33 46
25 38
9 13
9 22
29 31
35 40
6 18
32 36
20 28
24 28
0 41
6 36
5 38
11 15
17 26
25 40
17 31
9 34
28 48
10 33
35 38
12 18
17 27
8 46
16 46
10 32
33 35
17 34
42 47
4 42
3 4
35 45
4 27
10 14
5 15
13 21
2 3
5 12
2 24
10 44
12 36
1 42
18 22
12 26
18 32
6 24
23 34
16 25
17 41
26 27
3 43
0 14
2 8
30 41
18 28
15 33
16 40
13 41
7 20
44 46
15 45
15 27
34 36
14 42
4 30
9 24
12 41
5 44
15 38
35 42
6 38
3 15
17 22
4 22
5 30
8 10
3 37
4 8
26 44
9 28
28 34
9 39
16 44
25 41
8 21
13 48
3 7
2 32
13 46
9 41
12 44
13 39
1 43
25 32
18 45
11 29
28 31
10 38
2 38
12 14
6 47
29 48
0 37
10 40
30 43
20 37
8 41
19 26
7 44
24 31
8 20
25 42
20 24
0 21
16 21
0 33
28 32
45 46
5 36
33 39
8 40
2 12
1 5
13 44
22 42
38 40
19 47
28 37
36 42
0 9
35 43
20 23
10 39
0 16
3 44
22 40
8 37
17 40
7 43
15 22
15 40
4 12
13 27
33 44
15 31
8 27
12 31
17 33
31 47
32 38
11 25
12 34
9 48
18 38
20 30
34 48
13 42
29 44
16 48
26 37
14 18
12 30
6 46
4 32
30 36
5 10
6 9
0 1
7 35
15 43
13 31
32 47
22 23
23 48
33 40
17 38
14 17
7 10
25 30
27 35
12 15
31 42
18 37
12 32
33 34
4 47
36 37
9 38
11 34
16 41
19 25
7 21
5 24
15 48
6 10
11 13
3 47
5 28
8 16
3 30
14 25
12 27
5 35
19 29
25 36
4 16
4 21
14 29
27 44
16 17
21 36
20 25
29 33
15 26
5 40
7 28
13 16
30 33
28 45
17 18
1 40
25 47
11 32
14 33
3 23
13 34
36 46
43 45
6 35
12 40
8 9
24 37
13 18
6 12
3 31
9 14
6 30
19 44
14 34
20 41
26 33
17 48
6 41
10 48
9 21
1 17
0 48
18 34
33 43
3 27
0 31
11 36
1 36
37 45
27 31
7 42
10 13
39 41
19 20
21 37
31 34
1 27
32 35
29 32
22 37
8 28
1 38
30 31
7 36
22 39
2 16
17 36
34 42
3 42
35 39
10 28
21 24
11 40
21 28
14 16
36 43
20 45
5 16
6 44
30 35
4 15
26 31
15 35
24 35
14 30
29 46
19 33
10 31
19 39
11 23
21 47
1 29
12 45
0 4
18 20
19 32
12 39
20 38
24 40
15 28
22 30
14 46
35 37
13 17
20 34
1 47
9 40
21 41
46 47
36 39
17 32
21 34
6 48
6 26
16 38
15 47
7 14
22 25
23 47
4 26
22 35
15 19
20 29
20 39
9 11
22 43
37 42
8 32
23 25
23 46
5 41
12 42
20 33
31 35
18 30
12 23
20 32
7 9
10 46
12 16
33 47
35 47
11 43
24 42
6 40
4 43
2 18
45 48
2 15
27 28
40 46
36 48
12 47
5 31
27 42
6 20
9 20
1 2
4 18
22 41
37 44
0 13
4 39
17 35
26 43
36 44
20 43
30 47
15 36
9 43
7 27
4 13
27 43
18 41
25 48
11 47
18 26
21 35
32 46
4 7
