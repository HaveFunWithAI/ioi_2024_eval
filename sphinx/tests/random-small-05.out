49 380
24 20 0 0 15 32 39 34 32 14 41 37 38 27 3 48 7 6 35 18 23 33 14 14 9 32 20 30 10 32 24 44 11 47 36 4 23 33 23 34 38 39 4 37 3 6 40 7 20
